pkg/machines/libvirt-common.js

```diff
@@ -514,31 +502,6 @@ export function parseOsInfoList(dispatch, osList) {
     dispatch(updateOsInfoList(parsedList));
 }
 
-export function parseStoragePoolDumpxml(connectionName, storagePoolXml, id_overwrite) {
```

Comment:
summary from IRC discussion: I see why you want to eliminate this function, and pass around the original structure instead. I. e. knowledge of the structure would move from this function into the callers, in this spirit: `storagePool.type` → `storagePool.xml.pool._attributes.type`.

However, I'm not convinced that introducing a new external dependency for this, and converting it back and forth twice will actually help with robustness (aside from possible bugs in xml-js, and impedance mismatches between the conversions). Does it work to just pass around the original DOMParser object instead?

Reply:
Not using xml-js anymore